doc/internal/man3/OSSL_METHOD_STORE.pod

```diff
@@ -0,0 +1,101 @@
+=pod
+
+=head1 NAME
+
+ossl_method_store_new, ossl_method_store_free, ossl_method_store_init,
+ossl_method_store_cleanup, ossl_method_store_add, ossl_method_store_remove,
+ossl_method_store_fetch, ossl_method_store_set_global_properties,
+ossl_method_store_cache_get and ossl_method_store_cache_set
+- implementation method store and query
+
+=head1 SYNOPSIS
+
+ #include "internal/property.h"
+
+ typedef struct ossl_method_store_st OSSL_METHOD_STORE;
+
+ OSSL_METHOD_STORE *ossl_method_store_new(void);
+ void ossl_method_store_free(OSSL_METHOD_STORE *store);
+ int ossl_method_store_init(void);
+ void ossl_method_store_cleanup(void);
+ int ossl_method_store_add(OSSL_METHOD_STORE *store,
+                           int nid, const char *properties,
+                           void *implementation,
+                           void (*implementation_destruct)(void *));
+ int ossl_method_store_remove(OSSL_METHOD_STORE *store,
+                              int nid, const void *implementation);
+ int ossl_method_store_fetch(OSSL_METHOD_STORE *store,
+                             int nid, const char *properties,
+                             void **result);
+ int ossl_method_store_set_global_properties(OSSL_METHOD_STORE *store,
+                                            const char *prop_query);
+ int ossl_method_store_cache_get(OSSL_METHOD_STORE *store, int nid,
+                                 const char *prop_query, void **result);
+ int ossl_method_store_cache_set(OSSL_METHOD_STORE *store, int nid,
+                                 const char *prop_query, void *result);
+
+=head1 DESCRIPTION
+
+OSSL_METHOD_STORE stores implementations of algorithms that can be queried using
+properties and a NID.
+
+ossl_method_store_init() initialises the implementation method store subsystem.
+
+ossl_method_store_cleanup() cleans up and shuts down the implementation method
+store subsystem
+
+ossl_method_store_new() create a new empty implementation method store.
+
+ossl_method_store_free() frees resources allocated to B<store>.
+
+ossl_method_store_add() adds the B<implementation> to the B<store> as an
+instance of the algorithm indicated by B<nid> and the property definition
+<properties>. The optional B<implementation_destruct> function is called when
```

Comment:
May I suggest breaking the line after each end of sentence (i.e. the full stop)?  This is to avoid the dispute of one vs two spaces after a full stop, and lets local conventions decide instead (it has been observed that different installations have different conventions, but those will only be applied on a linebreak within a paragraph)

Reply:
I try to apply what I just wrote on any *new* POD file I write (or old ones if I rewrite them entirely).  Doing this on all existing POD files is a different story.

Reply:
Done.